src/main/web_src/src/components/bestilling/gjenopprett/GjenopprettModal.tsx

```diff
@@ -0,0 +1,68 @@
+import { useSelector } from 'react-redux'
+import DollyModal from '~/components/ui/modal/DollyModal'
+import { TitleValue } from '~/components/ui/titleValue/TitleValue'
+import Formatters from '~/utils/DataFormatter'
+import { Formik } from 'formik'
+import { filterMiljoe } from '~/components/miljoVelger/MiljoeInfo/TilgjengeligeMiljoer'
+import React, { Fragment } from 'react'
+import { MiljoVelger } from '~/components/miljoVelger/MiljoVelger'
+import NavButton from '~/components/ui/button/NavButton/NavButton'
+import * as yup from 'yup'
+import { ErrorBoundary } from '~/components/ui/appError/ErrorBoundary'
+
+type GjenopprettModal = {
+	gjenopprettHeader: any
+	environments?: Array<string>
+	submitFormik: any
+	closeModal: any
+	bestilling?: any
+}
+
+export const GjenopprettModal = ({
+	gjenopprettHeader,
+	environments,
+	submitFormik,
+	closeModal,
+	bestilling
+}: GjenopprettModal) => {
+	const tilgjengeligeEnvironments = useSelector((state: any) => state.environments.data)
+
+	const schemaValidation = yup.object().shape({
+		environments: yup.array().required('Velg minst ett miljø')
+	})
+
+	return (
+		<DollyModal isOpen={true} closeModal={closeModal} width="25%" overflow="auto">
```

Comment:
Kunne det vært fint med litt bredere modal? Dette er kanskje det vi har hatt den på gjenopprett fram til nå, men nå synes jeg den ser veldig smal og lang ut. Tror de med mindre skjermer må scrolle en del for å komme igjennom hele 😅

Reply:
I så fall kan forresten også size på TitleValue for Bestilt miljø settes litt bredere, så sparer vi noen linjer om det er mange miljøer 😄